force-app/main/default/classes/TestDuplicateEpisode.cls

```diff
@@ -0,0 +1,99 @@
+@isTest
+public with sharing class TestDuplicateEpisode {
+
+    @isTest
+    static void testAddSameEpisodes() {
+        
+        TV_Serie__c tvSerie = TestDataFactory.getSimpleTvSerie('serie1','www.google.pl','sfsdfds');
+        insert tvSerie;
+        Season__c seasonOne = TestDataFactory.getSimpleSeason('name', tvSerie.Id);
+      	insert seasonOne;
+        Episode__c episodeOne = TestDataFactory.getSimpleEpisode('name',seasonOne.Id, 'dfsdf');
+        insert episodeOne;
+        Episode__c episodeTwo = TestDataFactory.getSimpleEpisode('name',seasonOne.Id, 'dfsdf');
+        
+        String errorM = '';
+        Test.startTest();
+        try {
+        insert episodeTwo;         
+            }
+        catch(Exception e){
+            errorM = e.getMessage();
+        }
+        Test.stopTest();
+        System.assertNotEquals('', errorM);
```

Comment:
If insert fails the error message should be as you set it in trigger ('here is already an episode of the same name this season!')

Reply:
Done